code/modules/cargo/console.dm

```diff
@@ -152,6 +153,11 @@ proc/get_cost_multiplier(var/datum/planet/PL)
 						"cost" = price,// / cost_mult,
 						"id" = "\ref[O]"
 					))
+				data["deliver"] += list(list(

```

Comment:
Volatile Bombs won't reach here because of the continue on line 143. Might want to also make deliver only show for mission items.